player.py

```diff
@@ -143,7 +144,7 @@ def __init__(
         self.rip_duration = 12 + 4 * rip_glyph + 4 * t7_2p
         self.rake_duration = 9 + 3 * t9_2p
         self.lacerate_multi = 1 + 0.05 * t7_2p
```

Comment:
2pT10 boosts both the initial and periodic damage of Lacerate, so it should show up here as well.

Reply:
Verified in wowhead, thanks